lib/tasks/import.rake

```diff
@@ -1,24 +1,20 @@
 # frozen_string_literal: true
 
-require "tasks/whitehall_news_importer"
+require "tasks/whitehall_importer"
```

Comment:
Minor, but since https://github.com/alphagov/content-publisher/commit/d2d82b9ea4b3de2081a5c473005bbd6f867817b0 we can drop this require (same for https://github.com/alphagov/content-publisher/pull/1303/files#diff-88c72fb190dd2a4c9d8510058c97382bR3)